Control coverage/Feature/CDN.md

```diff
@@ -62,3 +62,52 @@ One of the following conditions is met:
 properties.isHttpAllowed, properties.isHttpsAllowed, properties.deliveryPolicy.rules
  <br />
 
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_FrontDoor_CDNProfile_DP_Use_Secure_TLS_Version_Trial 
+
+### Display Name 
+[Trial] Front Door should have Approved Minimum TLS version
```

Comment:
start approved and minimum with lower case.

Reply:
Fixed